<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TATRC-17323" name="testCases.Admin.Metadata.MetadataCategoryAddEditDelete" tests="1" failures="1" timestamp="07 Nov 2019 13:29:04 GMT" time="84.909" errors="0">
  <testcase name="execute" time="84.909" classname="testCases.Admin.Metadata.MetadataCategoryAddEditDelete">
    <failure type="java.lang.AssertionError" message="Validate URL changed to https://nat-p12.tatrc.org/JMedSim_Test/MetadataCategories/Index.* expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Validate URL changed to https://nat-p12.tatrc.org/JMedSim_Test/MetadataCategories/Index.* expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at pageFactories.MenusPageFactory.<init>(MenusPageFactory.java:38)
	at pageFactories.IndexBase.<init>(IndexBase.java:30)
	at pageFactories.Admin.AdminMetadataPages.MetadataCategoryPageFactory.<init>(MetadataCategoryPageFactory.java:28)
	at testCases.Admin.Metadata.MetadataCategoryAddEditDelete.editNewMetadataCategoryRecord(MetadataCategoryAddEditDelete.java:155)
	at testCases.Admin.Metadata.MetadataCategoryAddEditDelete.execute(MetadataCategoryAddEditDelete.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- execute -->
</testsuite> <!-- testCases.Admin.Metadata.MetadataCategoryAddEditDelete -->
